### DISCUSSION TITLE
can‘t through a project's  detail  stars(who stars this project)  when login github

### DISCUSSION THREAD

**Discussion by ningmingxiao** [community/community#8777] [Other Feature Feedback, Questions, & Ideas]
when login github.com i can‘t through a project's star now, only when i login out i can through it.

**Comment by brianlovin** [comment]
Thanks for the note @ningmingxiao — the entry point to view a list of people who have starred a repo has been moved to the repository sidebar:

![Screen Shot 2022-01-04 at 17 38 10@2x](https://user-images.githubusercontent.com/1923260/148146990-3352b419-0152-450e-a327-b56f09d62b95.png)
